common/annotations/my.xml

```diff
@@ -1933,7 +1933,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<annotation cp="🚏" type="tts">ဘတ်စ်ကား မှတ်တိုင်</annotation>
 		<annotation cp="🛣">ကားလမ်း | ဟိုင်းဝေး | အမြန်လမ်း</annotation>
 		<annotation cp="🛣" type="tts">ကားလမ်း</annotation>
-		<annotation cp="🛤">သံလမှး သံလမှး၊ ရထှား သံလမှး သံလမှး သံလမှး သံလမွှး သံလမွျး သံလမျး | သံလမ်း</annotation>
+		<annotation cp="🛤">သံလမှး သံလမှး၊ ရထှား သံလမှး သံလမှး သံလမှး သံလမှး သံလမြှး သံလမြျး သံလမျး</annotation>
```

Comment:
These are not good either.  All of them are garbled although I think this is more like "a bunch of unicode characters being randomly put together" kind of incorrect.